environment.yml

```diff
@@ -13,8 +13,8 @@ dependencies:
     - jupyter-sshd-proxy
     - jupyterlab-bxplorer
 variables:
-  TITILER_STAC_ENDPOINT: 'https://staging-stac.delta-backend.com'
-  TITILER_ENDPOINT: 'https://staging-raster.delta-backend.com'
+  TITILER_STAC_ENDPOINT: 'https://openveda.cloud/api/stac'
+  TITILER_ENDPOINT: 'https://openveda.cloud/api/raster'
   STAC_CATALOG_NAME: 'VEDA STAC'
-  STAC_CATALOG_URL: 'https://staging-stac.delta-backend.com/'
+  STAC_CATALOG_URL: 'https://openveda.cloud/api/stac'
   STAC_BROWSER_URL: 'http://veda-staging-stac-browser.s3-website-us-west-2.amazonaws.com/'
```

Comment:
@anayeaye is there a replacement for http://veda-staging-stac-browser.s3-website-us-west-2.amazonaws.com/?

Reply:
Yep, the production browser is at https://openveda.cloud